src/current/v23.2/work-with-virtual-clusters.md

```diff
@@ -0,0 +1,197 @@
+
+## Configure cluster settings
+
+When [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}) is enabled, each [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) has a scope, which may be the virtual cluster or the system interface.
+
+- When a cluster setting is scoped to the virtual cluster, it affects only the virtual cluster and not the system interface. To configure a cluster setting that is scoped to the virtual cluster, you must have the `admin` role on the virtual cluster, and you must connect to the virtual cluster before configuring the setting. The majority of cluster settings are scoped to the virtual cluster and are visible only when connected to the virtual cluster.
+- When a cluster setting is scoped to the system interface, it effects the entire storage cluster. To configure a cluster setting that is scoped to the system interface, you must have the `admin` role on the system interface, and you must connect to the system interface before configuring the setting.
+- When a cluster setting is system-visible, it can be set only from the system interface but can be queried from any virtual cluster. For example, virtual cluster can query a system-visible cluster setting's value to help adapt to the storage cluster's configuration.
+
+For more details, including the scope of each cluster setting, refer to [Cluster Setting Scopes with Cluster Virtualization enabled]({% link {{ page.version.version }}/cluster-virtualization-setting-scopes.md %}).
+
+## Upgrade a cluster
+
+When cluster virtualization is enabled to upgrade to a new major version, you must:
+
+1. Replace the binary on each node and restart the node.
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade on the system interface to upgrade it, or roll back the upgrade if you decide not to finalize it. Until it is finalized, the cluster still operates in compatibility with the previous major version, and virtual clusters cannot be upgraded.
+1. After the system interface is finalized, finalize the upgrade on the virtual cluster to upgrade it, or roll it back if you decide not to finalize it. Until it is finalized, the virtual cluster still operates in compatibility with the previous major version, some features may not be available on the virtual cluster.
+
+This allows you to roll back an upgrade of the system interface without impacting schemas or data in virtual clusters. The system interface can be at most one major version ahead of virtual clusters. For example, when v24.1 is released, a system interface on CockroachDB v24.1 can have virtual clusters on CockroachDB v23.2.
+
+To apply a patch-version upgrade, you must only replace the binary on each node and restart the node. Finalization is not required. It is not possible for a virtual cluster to run a different patch version than the system interface.
```

Comment:
```suggestion
To apply a patch-version upgrade, you must only replace the binary on each node and restart the node. Finalization is not required. 
```

I'd remove the last part.  They're running in the same binary, so this is somewhat implied.